### ISSUE TITLE
Pagination directive needs roles for accessibility

### ISSUE THREAD

**Issue by dadoadk** [angular-ui/bootstrap#6383]
### Bug description:
Missing role="menu" on `<ul>` and role="menuitem" on `<li>` elements

Bootstrap:
